upgrade_test.py

```diff
@@ -244,7 +244,8 @@ def upgrade_node(self, node, upgrade_sstables=True):
         check_reload_systemd_config(node)
         # Current default 300s aren't enough for upgrade test of Debian 9.
         # Related issue: https://github.com/scylladb/scylla-cluster-tests/issues/1726
-        node.start_scylla_server(verify_up_timeout=500)
+        if not node.run_scylla_sysconfig_setup():
```

Comment:
`run_scylla_sysconfig_setup()` is not returning anything, so it looks it will always start scylla server.

Reply:
Looks like I forgot to unstash the changes there. Anyway, this is now simplified - we need to restart anyway, as per the upgrade procedure and the `run_scylla_sysconfig_setup()` method is now not doing any restarts on its own.